application/account-management/Api/appsettings.development.json

```diff
@@ -7,6 +7,6 @@
     }
   },
   "ConnectionStrings": {
-    "Default": "Server=${SQL_SERVER_NAME};Database=account-management;User Id=sa;Password=${SQL_SERVER_PASSWORD};Encrypt=False;TrustServerCertificate=True"
+    "Default": "Server=localhost;Database=account-management;Encrypt=False;TrustServerCertificate=True"
```

Comment:
I believe we can delete this. As it's now pulling the connecting string called "account-management".

Reply:
Removed. I agree. It's not needed.